### DISCUSSION TITLE
Send database notifications

### DISCUSSION THREAD

**Discussion by newtec-foggia** [filamentphp/filament#10708] [Help]
Can I use database notifications within a job?

the notification works inside an action but I can't get it to work inside a job

![image](https://github.com/filamentphp/filament/assets/83398776/f6ce1a15-4e1b-4fa2-b087-c3a5db99af9c)

**Comment by domthomas-dev** [comment]
Hi, I think you have a conception problem. In a job,`auth()->user()` is null.
